app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsDetailsActions/PipelineDetailsActionsButton/index.tsx

```diff
@@ -370,9 +407,15 @@ export default class PipelineDetailsActionsButton extends Component {
               {T.translate(`${PREFIX}.export`)}
             </li>
             {this.props.sourceControlManagementEnabled && (
-              <li role="button" onClick={this.toggleCommitModal} data-testid="push-pipeline">
-                {T.translate('features.SourceControlManagement.push.pushButton')}
-              </li>
+              <div>
+                <hr />
+                <li role="button" onClick={this.toggleCommitModal} data-testid="push-pipeline">
+                  {T.translate(`${SCM_PREFIX}.push.pushButton`)}
```

Comment:
wondering if it's slightly better to use `push to git` instead of `push to remote`?